src/style/document-header.less

```diff
@@ -59,13 +59,54 @@
 .document-header {
     height: @toolbar-button-size;
     line-height: @toolbar-button-size;
-    max-width: 100%;
-    width: 100%;
+    flex-grow: 10;
     position: relative;
     overflow: hidden;
     display: flex;
 }
 
+.generator-header {
+    height: @toolbar-button-size;
+    line-height: @toolbar-button-size;
+    max-width: 5rem;
+    width: 5rem;
+    display: flex;
+    flex-grow: 1;
+}
+
+.generator-header-buttons { 
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.generator-header-buttons .button-simple.button-plus {
+        margin-right: 2rem;
+
+        & svg:hover {
+            fill: @item;
+        }
+
+        &.button-simple__disabled {
+            fill: @item-inactive;
+
+            &:hover {
+                fill: @item-inactive;
+            }
+        }
+}
+
+.generator-header-buttons .button-simple.button-simple__disabled.button-plus svg {
```

Comment:
sweet mother of mercy that's a lot of selectors.  It seems like we should have a far simpler way to make the SVG disabled for disabled buttons.  just sayin.  maybe a future cleanup task.